README.md

```diff
@@ -1118,6 +1118,8 @@ See also [About Rust’s Machine Learning Community](https://medium.com/@autumn_
 ### Filesystem
 
 [[filesystem](https://crates.io/keywords/filesystem)]
+* Libraries
+  * [jonhkr/rust-file-seq](https://github.com/jonhkr/rust-file-seq) -> Fail-safe sequence implementation that uses the file system as store
```

Comment:
Please add the CI badge as per https://github.com/rust-unofficial/awesome-rust/blob/master/CONTRIBUTING.md